base_jsonify/models/models.py

```diff
@@ -63,6 +64,8 @@ def jsonify(self, parser):
                 else:
                     res[json_key] = rec._jsonify_value(field_name)
             result.append(res)
+        if one:
```

Comment:
Wouldn't you add a `self.ensure_one()` here? 
Otherwise a call with multiple records and a `one=True` seems very dubious?
(and I agree, I almost added a jsonify_one() method)

Reply:
good catch, done, tnx!